app/views/passengers/show.haml

```diff
@@ -1,14 +1,32 @@
 %h1= @passenger.name
 %p
   %b Address:
-  =@passenger.address
+  = @passenger.address
+%p
+  %b Spire:
+  = @passenger.spire
 %p
   %b Email:
   = @passenger.email
 %p
   %b Phone:
   = @passenger.phone
 %p
+  %b UMass Status:
+  = @passenger.status
+%p
+  %b Registered with disability services?
+  - if @passenger.registered_with_disability_services?
+    Yes
+  - else
+    No
```

Comment:
It might be nice to translate the `if boolean then 'Yes' else 'No'` into a helper method, or just use the existing one for the cross and check images.